src/actions/clientActions.js

```diff
@@ -0,0 +1,22 @@
+import { CLIENT_CHANGE } from '../constants/actionTypes'
+
+function isSmallScreen () {
+  return global.window.innerWidth <= 768
+}
+
+function clientChange (smallScreen) {
+  return {
+    type: CLIENT_CHANGE,
```

Comment:
Yes – but! Think about using Fittable in other contexts, e.g. hybrid mobile application (e.g. PhoneGap), being rendered on server (i.e. isomorphic app) or being used in offline context. IMO the definititon of UA then becomes too wide, thus the naming `client` as a runtime environment for Fittable.